### PR TITLE
Define "hub server"

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/turt2live/ietf-mimi-terminology/issues/2

### CODE REVIEW

File: draft-ralston-mimi-terminology.md [hunk 1]
```diff
@@ -152,6 +152,24 @@ in the same logical place as a client (i.e.: in peer-to-peer environments), howe
...
+shortened to "owner" to avoid confusion with other definitions in this document.
+
+**Server Owner**: The entity or person responsible for administrative function of the server,
+such as creating user accounts, banning/blocking chats, etc. This should NOT be shortened to
+"owner" to avoid confusion with other definitions in this document.
+
+**Server Operator**: The entity or person responsible for uptime and availability of a given
+server. This is typically the same as the server owner, though servers may be deployed to a
+shared hosting environment. The hosting provider becomes the server operator while the account
+holder with that provider becomes the server owner.
```
Comment (turt2live) [inline draft-ralston-mimi-terminology.md:171]: It's fairly unlikely that hosting providers for MIMI implementations/transports will sprout up (it just doesn't make sense), but it feels distinct enough from server owners to be worth writing down.

**Comment by turt2live** [comment]
@coopdanger please take a look - these definitions are potentially controversial in nature

**Comment by coopdanger** [comment]
As I've thought about this, I wonder if at this point we need only the definition of hub server. Are you using the server owner and server operator definitions anywhere? If not, I think defining them creates confusion and we can avoid it by not defining them.

Regarding owning server, I think we can defer trying to write a definition until we've had a detailed discussion about how the authorization policy for a chat is going to be decided between the participating providers and their servers.

**Comment by turt2live** [comment]
> As I've thought about this, I wonder if at this point we need only the definition of hub server. Are you using the server owner and server operator definitions anywhere? If not, I think defining them creates confusion and we can avoid it by not defining them.

I don't think we're using them anywhere, and I did consider not writing definitions for them. It felt a bit awkward to say "don't shorten this to 'owner' because of undefined terms", but we might be able to get around that for now.



> Regarding owning server, I think we can defer trying to write a definition until we've had a detailed discussion about how the authorization policy for a chat is going to be decided between the participating providers and their servers.

fair, will remove it for now.


> Also does this imply that the protocol needs to support cases where there is no hub server? Because I thought we had pretty good agreement that there would always be a hub server.

This document was drafted when we didn't have consensus on design, yea. I'll add some words to fix that up in this PR.

**Comment by turt2live** [comment]
@coopdanger I think this looks better now? I'll open issues for the definitions I removed from this PR so we have a base to work from if needed.

**Comment by coopdanger** [comment]
👍
